src/utils/flags.ts

```diff
@@ -157,4 +159,8 @@ export const flags: FlagMap = {
 	noSnapshot: {
 		description: noSnapshotDescription,
 	},
+	version: {
```

Comment:
this is conflicting with `lisk -v`

Reply:
i think `version` is also conflicting =(

Reply:
`release`? or `lisk-version`?